README.asc

```diff
@@ -1,32 +1,22 @@
 = Pro Git, Segunda Edição, Português do Brasil
 
-Bem-vindo à segunda edição do livro Pro Git
+Bem-vindo à segunda edição do livro Pro Git.
 
 > If you're looking for official version, in English, see https://github.com/progit/progit2[].
 
 Você pode achar esse livro online em: http://git-scm.com/book
 
-Como a primeira, a segunda edição do livro Pro Git é open source sob uma licença Creative Commons.
+Como na primeira edição, a segunda edição do livro Pro Git é open source sob uma licença Creative Commons.
 
-Algumas coisas mudaram desde quando a primeira edição foi disponibilizada de forma livre.
-Umas delas foi que mudamos o texto do livro de Markdown para o maravilhoso formato Asciidoc. Nós também passamos a usar a https://atlas.oreilly.com[Plataforma Atlas] da O'Reilly para a geração contínua dos livros, assim os formatos mais conhecidos estão sempre disponíves em todas os idiomas.
+Algumas coisas mudaram desde a liberação do código da primeira edição.
+Umas delas, nós passamos o texto do livro de Markdown para o maravilhoso formato Asciidoc.
 
-Nós também passamos a manter as traduções em repositórios separados em vez usar de subdiretórios do repositório original em inglês.
-Veja link:CONTRIBUTING.md[o documento para contribuições] para mais informações.
+Nós também passamos a manter as traduções em repositórios separados ao invés de usar subdiretórios do repositório em inglês.
+Veja link:TRANSLATING.md[o documento de tradução] para mais informações.
 
 == Como gerar o livro
 
-Existem algumas formas para gerar o e-book a partir deste código fonte.
-
-O mais fácil é simplesmente deixar que a gente faça isso.
-Um robô fica esperando por mudanças no branch principal e automaticamente gera o livro para todos.
-
-Você acha as versões atuais em http://git-scm.com/book[] e mais informações sobre a geração dos livros estão disponíveis em https://progit.org[].
-
-=== Manualmente
-
-Outra forma de gerar os arquivos do ebooks é de forma manual, usando o Asciidoctor.
-Se você executar os comandos a seguir você _deve_ obter os arquivos em HTML, Epub, Mobi e PDF:
+Você pode gerar os arquivos de e-book manualmente com o Asciidoctor. Se você executar o seguinte, você _pode_ atualmente obter arquivos de saída em HTML, Epub, Mobi e PDF:
```

Comment:
The correct translation of "actually" to pt-br would be something like "realmente", or "na verdade." Translating that to "atualmente" is a case of [false friends](https://englishlive.ef.com/pt-br/blog/false-friends-falsos-cognatos-em-ingles-parece-mas-nao-e/).

As soon as this is fixed, I can merge. Thanks again for your help and sorry for the delay.